project-mappings

```diff
@@ -2,14 +2,16 @@
 :beam-vendor-sdks-java-extensions-protobuf :vendor:sdks-java-extensions-protobuf
 :beam-vendor-guava-20_0 :vendor:guava-20_0
 :beam-vendor-grpc-1_13_1 :vendor:grpc-1_13_1
-:beam-sdks-python-test-suites-tox-py36 :sdks:python:test-suites:tox:py37
-:beam-sdks-python-test-suites-tox-py36 :sdks:python:test-suites:tox:py36
-:beam-sdks-python-test-suites-tox-py35 :sdks:python:test-suites:tox:py35
 :beam-sdks-python-test-suites-direct-py35 :sdks:python:test-suites:direct:py35
 :beam-sdks-python-test-suites-direct-py36 :sdks:python:test-suites:direct:py36
+:beam-sdks-python-test-suites-direct-py37 :sdks:python:test-suites:direct:py37
```

Comment:
Do we even need to to add this here?

Reply:
It is a temporary alias to help people transition from old ways to name gradle task to new names, see: https://github.com/apache/beam/blob/abece47cc1c1c88a519e54e67a2d358b439cf69c/deprecation-warning.txt